### PR TITLE
KHR_audio - fix readme

### PR DESCRIPTION
**Pull request by antpb**
This PR fixes the readme, more details to come.

### CODE REVIEW

File: extensions/2.0/KHR_audio/README.md [hunk 1]
```diff
@@ -262,6 +276,7 @@ Audio emitters of type `positional` may be added to nodes using the following sy
                     "emitter": 2
                 }
             }
+            ]
```
Comment (aaronfranke) [inline extensions/2.0/KHR_audio/README.md:279]: Stray closing bracket.

Please run all JSON through a formatter before pushing :)

File: extensions/2.0/KHR_audio/README.md [hunk 2]
```diff
@@ -243,7 +249,15 @@ Audio emitters of type `global` may be added to scenes using the following synta
         {
             "extensions": {
                 "KHR_audio": {
-                    "emitters": [0, 1]
+                    "emitters": [
```
Comment (aaronfranke) [inline extensions/2.0/KHR_audio/README.md:252]: https://github.com/omigroup/gltf-extensions/blob/main/extensions/2.0/KHR_audio/schema/scene.KHR_audio.schema.json#L9

This is not correct, these are supposed to be indices.

> "description": "The indices of each global audio emitter. Positional audio emitters may not be added to the scene node.",

**Review by aaronfranke — APPROVED** [review]
A good update, this will unblock me from building a Godot implementation.

**Comment by antpb** [comment]
Screenshot 2023-10-12 at 5 46 23 PM
Screenshot 2023-10-12 at 5 46 18 PM
Screenshot 2023-10-12 at 5 46 12 PM

**Pull request merged**